src/pybind/cephfs/cephfs.pyx

```diff
@@ -695,16 +744,17 @@ cdef class LibCephFS(object):
         path = cstr(path, 'path')
         cdef:
             char* _path = path
-            ceph_dir_result *dir_handler
+            ceph_dir_result* handle
```

Comment:
Please change the parameter name in docstrings too.

Reply:
good catch!